src/heads-up/views/slack.vue

```diff
@@ -44,8 +94,33 @@
             </p>
           </dd>
         </template>
-        <template v-if="og.description">
-          <dt>og:description</dt>
+        <template>
+          <dt>
+            <p v-if="!og.description">
+              og:description
+            </p>
+            <app-tooltip
+              class="properties-item__tooltip"
+              placement="bottom-start"
+            >
+              <InfoIcon
+                v-if="og.description === null"
```

Comment:
Same for this, can this not be `!og.description`?

Reply:
In this example it can be `!og.description`